dashboard2/settings/jptt.md

```diff
@@ -164,3 +164,21 @@ You can also comment out elements by adding `#` at the start of a line. In this
 "%"."exampleObject"."nestedObject"."sentence04".! skip
 "%"."exampleObject"."exampleArray".1."value" skip
 ```
+
+### JavaScript template literals
+
+Template literals are similar to strings but they support inline variables. They are surrounded by backtick (`) characters. They are different from regular strings in that they support inline variables and expressions. For example, 
+
+```
+let publishedText = “Published on “ + date;
+```
+
+could be expressed as
+
+```
+let publishedText = `Published on ${date}`;
+```
+
+Simple cases like the above (or literals without variables) are supported. The expression will become a tag in the Workbench and in XLIFF exports. JavaScript allows any expression to be added to literals but we recommend you to stick to simple expressions if possible.
```

Comment:
Does that mean function invocations are not supported at all? If so, that should be mentioned.

Reply:
Well, it did work with

```js
let cica = () => {
    return "cica";
}

let output = `Ez a ${cica()} nagyon aranyos`;

console.log(output);
```

I just don't want to suggest they put something horribly complex in there.

Reply:
Well, that's good so far. What was the path to get `"cica"` extracted here?

Reply:
It's just `cica` and it works as expected.